src/screens/FeedScreen/Header.js

```diff
@@ -227,20 +325,47 @@ const _renderProfileNormal = ({
           <GlobalButton
             onPress={navigateToProfile}
             style={[styles.containerFeedProfile, {paddingBottom: 5}]}>
-            <View style={[styles.containerFeedName, {alignItems: 'flex-end'}]}>
-              <Text style={styles.feedUsername}>
-                {username || StringConstant.feedDeletedUserName}
-              </Text>
+            <View>
+              <View
+                style={[
+                  {
+                    flexDirection: 'row',
+                    alignItems: 'center'
+                  }
+                ]}>
+                <View
+                  style={{
+                    flexDirection: 'row'
+                  }}>
+                  <Text>{username || StringConstant.feedDeletedUserName}</Text>
+                  {!isSelf && (
+                    <React.Fragment>
+                      <View style={styles.point} />
+                      <TouchableOpacity onPress={() => onPressFollUnFoll(isFollow)}>
+                        <Text style={isFollow ? styles.textFollowing : styles.textFollow}>
+                          {isFollow ? 'Following' : 'Follow'}
+                        </Text>
+                      </TouchableOpacity>
+                    </React.Fragment>
+                  )}
+                </View>
 
-              <GlobalButton
-                buttonStyle={{marginLeft: 'auto', paddingBottom: 0, alignSelf: 'center'}}
-                onPress={onHeaderOptionClicked}>
-                {hideThreeDot ? null : (
-                  <View style={{zIndex: 1000}}>
-                    <ElipsisIcon width={4} height={14} fill={COLORS.blackgrey} />
-                  </View>
-                )}
-              </GlobalButton>
+                <GlobalButton
+                  buttonStyle={{
+                    marginLeft: 'auto',
+                    paddingBottom: 0,
+                    alignSelf: 'center'
+                  }}
+                  onPress={() => {
+                    refSheet.current.open();
+                  }}>
+                  {hideThreeDot ? null : (
+                    <View style={{zIndex: 1000}}>
+                      <ElipsisIcon width={4} height={14} fill={COLORS.blackgrey} />
+                    </View>
+                  )}
+                </GlobalButton>
+              </View>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to my previous comments, it would be better to check whether the `onPressFollUnFoll` function is provided before rendering the TouchableOpacity. This way, you can prevent potential errors if the button is clicked but no function was provided.
```diff
  {!isSelf && (
    <React.Fragment>
      <View style={styles.point} />
+     {onPressFollUnFoll && (
        <TouchableOpacity onPress={() => onPressFollUnFoll(isFollow)}>
          <Text style={isFollow ? styles.textFollowing : styles.textFollow}>
            {isFollow ? 'Following' : 'Follow'}
          </Text>
        </TouchableOpacity>
+     )}
    </React.Fragment>
  )}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->